modules/configure-an-additional-clientCA.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/api-server.adoc
+
+:_content-type: PROCEDURE
+[id="configure-an-additional-clientCA-for-the-OpenShift-API-server_{context}"]
+
+= Configure an additional clientCA for the OpenShift API server
+
+Optionally, you may choose to invalidate the installer-generated kubeconfig. You would do this when:
```

Comment:
```suggestion
Optionally, you can invalidate the installer-generated kubeconfig. You might do this if any of the following conditions exist:
```

Reply:
Is _invalidate_ the right word? I should think _replace_, which you use later is better. If you invalidate the kubeconfig, there is no kubeconfig, right?

Reply:
used suggested text, including "replace."